### ISSUE TITLE
Add deploy.xml file to make the deploy automatic using github-action

### ISSUE THREAD

**Issue by palashjhabak** [techlift-tech/TelegramBotForPokerAnalysis#27]

**Comment by disha-2401** [comment]
Isn't it a .yml file? I couldn't find documentation for a xml file.
I've checked GitHub actions documentation and there's details about a yml file for CI/CD, if there's any documentation about this provide me link

**Comment by palashjhabak** [comment]
@disha-2401  yes my bad its a .yml file

**Comment by disha-2401** [comment]
> @disha-2401 yes my bad its a .yml file
 
[deploy.txt](https://github.com/techlift-tech/TelegramBotForPokerAnalysis/files/6305530/deploy.txt)
I've created this file, check if it's ok then you can add this file to any branch or master branch.
.yml files are not supported to upload here so I've uploaded a text file instead.

**Comment by palashjhabak** [comment]
@disha-2401 yes the file seems fine. For the publish command we will use it with more parameters, let me share you a sample deploy.yml file once
